guide/concepts/_stop-start-restart-behaviour.camp.md

```diff
@@ -0,0 +1,56 @@
+## Top-level applications
+A top-level application is a grouping of other entities, pulling them together into the "application" of your choice. This could range from a single app-server, to an app that is a composite of a no-sql cluster (e.g. MongoDB sharded cluster, or Cassandra spread over multiple datacenters), a cluster of load-balanced app-servers, message brokers, etc.
```

Comment:
@algairim `pulling them` ? Is this intended? I don't understand where they are pulled from. Myabe rephrase. also replace n-sql with NoSQL, I think that is the correct acronym.

Reply:
This is an original text, it has been moved only.

To me, it looks like 'no-sql' is intended, emphasising a cluster that has nothing to do with SQL.

Reply:
As you wish.